src/content/learn/javascript-in-jsx-with-curly-braces.md

```diff
@@ -66,28 +66,28 @@ export default function Avatar() {
 
 </Sandpack>
 
-Primećujete razliku između `className="avatar"`, što određuje CSS klasu `"avatar"` koja čini sliku okruglom, i `src={avatar}` koja čita vrednost JavaScript promenljive koja se zove `avatar`. To je zato što vitičaste zagrade vam dozvoljavaju da radite sa JavaScript-om upravo tamo u vašem markup-u!
+Primećujete razliku između `className="avatar"`, što određuje CSS klasu `"avatar"` koja čini sliku okruglom, i `src={avatar}` koja čita vrednost JavaScript promenljive koja se zove `avatar`. To je zato što vam vitičaste zagrade dozvoljavaju da radite sa JavaScript-om upravo u vašem markup-u!
 
 ## Koristite vitičaste zagrade: Prozor u JavaScript svet {/*using-curly-braces-a-window-into-the-javascript-world*/}
 
-JSX je specijalan način pisanja JavaScript-a. To znači da je moguće koristiti JavaScript unutar njega - sa vitičastim zagradama `{ }`. Primer ispod prvo deklariše ime za naučnika, `ime`, a zatim ga ugnezduje(nest) sa vitičastim zagradama unutar `<h1>`:
+JSX je specijalan način pisanja JavaScript-a. To znači da je moguće koristiti JavaScript unutar njega - sa vitičastim zagradama `{ }`. Primer ispod prvo deklariše ime za naučnika, `name`, a zatim ga ubacuje vitičastim zagradama unutar `<h1>`:
 
 <Sandpack>
 
 ```js
 export default function TodoList() {
-  const ime = 'Gregorio Y. Zara';
+  const name = 'Gregorio Y. Zara';
```

Comment:
Since we left initial code on all other places, except translating comments, I've used english variable names here as well.

Reply:
I think that things like variable names should be translated; maybe naziv would be better than Ime.
Nevertheless, I will merge your current corrections, and we can discuss this topic later :D

Reply:
Please traverse through your changes before I merge them, as I noticed translations of open source and a few more. Please do the check again, fix this kind of detail, and push it again  :D